src/BMS.cpp

```diff
@@ -261,16 +261,20 @@ bool BMS::isHealthy() {
 
 void BMS::updateVoltageReadings() {
     // TODO: Handle when an error has taken place
-    bq.getCellVoltage(cellVoltage, &totalVoltage, &minCellVoltage, &minCellVoltageID, &maxCellVoltage, &maxCellVoltageID);
+    for(int8_t i = 0; i < DEV::BQ76952::NUM_CELLS; i++){
+        cellVoltage[i]= i;
+    }
+    bq.getCellVoltage(cellVoltage, totalVoltage, voltageInfo);
+    EVT::core::time::wait(10);
 }
 
 void BMS::clearVoltageReadings() {
-    totalVoltage = 0;
-
-    // Zero out each cell voltage
-    for (uint8_t i = 0; i < DEV::BQ76952::NUM_CELLS; i++) {
-        cellVoltage[i] = 0;
-    }
+//    totalVoltage = 0;
```

Comment:
Why is this commented out?

Reply:
I did that for testing purposes. It has been fixed.